src/stk/molecular/topology_graphs/topology_graph/optimizer/collapser.py

```diff
@@ -0,0 +1,177 @@
+"""
+Collapser
+=========
+
+"""
+
+from .optimizer import Optimizer
+from .utilities import (
+    get_mch_bond_topology,
+    merge_subunits,
+    OptimizationIncompleteError,
+)
+
+import mchammer as mch
+
+
+class Collapser(Optimizer):
+    """
+    Perform rigid-body collapse of molecule.
+
+    Implementation: [1]_
+
+    .. [1] https://github.com/andrewtarzia/MCHammer.
+
+    """
+
+    def __init__(
+        self,
+        step_size=0.1,
+        distance_threshold=1.5,
+        scale_steps=True,
+        save_trajectory=False,
+    ):
+        """
+        Initialize an instance of :class:`.Collapser`.
+
+        Parameters
+        ----------
+        step_size : :class:`float`, optional
+            The relative size of the step to take during collapse.
+            Defaults to 0.1 Angstrom.
+
+        distance_threshold : :class:`float`, optional
+            Distance between distinct subunits to use as
+            threshold for halting collapse in Angstrom.
+            Defaults to 1.5 Angstrom.
+
+        scale_steps : :class:`bool`, optional
+            Whether to scale the step of each distict building block
+            by their relative distance from the molecules centroid.
+            Defaults to ``True``
+
+        save_trajectory : :class:`bool`, optional
+            `True` to save Collapser trajectory and information in
+            :attr:`_trajectory_data`.
```

Comment:
Expose `_trajecotry_data` in some sort of public api, like a getter

Reply:
Done.